tests/parser/test_class2D.py

```diff
@@ -1,21 +1,41 @@
 import pytest
 import relion
+import pathlib
+import sys
 from pprint import pprint
 
 
 @pytest.fixture
-def class2d(dials_data):
-    return relion.Project(dials_data("relion_tutorial_data")).class2D
+def proj(dials_data):
+    return relion.Project(dials_data("relion_tutorial_data"))
+
+
+@pytest.fixture
+def class2d(proj):
+    return proj.class2D
 
 
 def test_list_all_jobs_in_class2d_directory(class2d):
     """
     When used in an iterator context the Class2D instance returns
-    all known job and alias names.
+    all known job (dropping alias names).
     """
     assert sorted(class2d) == ["job008", "job013"]
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="does not run on windows")
+def test_list_all_jobs_in_class2d_directory_symlink(proj):
+    """
+    Test that aliases are dropped so that jobs aren't double
+    counted when iterated over
+    """
```

Comment:
`def test_aliases_are_dropped_on_iterating_so_jobs_arent_double_counted(proj)`
?

Since you never need to type out test function names these can be fully descriptive. That way you already know what a test was supposed to do when you see the pytest failure list, and don't need to hunt a comment first.

(Same, or similar for the rest of the added tests)